st2common/st2common/models/db/keyvalue.py

```diff
@@ -37,6 +37,7 @@ class KeyValuePairDB(stormbase.StormBaseDB, stormbase.UIDFieldMixin):
     scope = me.StringField(default=FULL_SYSTEM_SCOPE, unique_with="name")
     name = me.StringField(required=True)
     value = me.StringField()
+    new_encrypted_value = me.StringField()
```

Comment:
Why is there a new field for the value?
I see where `new_encrypted_value` gets written in the migration script, but I don't see anything else that consumes it (eg to copy it to the `value` field after successfully reencrypting all the values).